plugins/npm/nexus-npm-repository-plugin/src/main/java/com/bolyuba/nexus/plugin/npm/hosted/NpmHostedRepository.java

```diff
@@ -25,4 +25,11 @@
 {
   @Override
   HostedMetadataService getMetadataService();
+
+  /**
+   * Recreates npm metadata from deployed tarballs. During this operation, repository should be read only.
```

Comment:
Basically I followed the pattern we have for method like expireCaches, etc. `true` means there were packages processed (db changed). `false` means no packages processed (db unchanged), basically means repository is empty, at least contains no tarballs. Am totally neutral is return value needed at all or not.